student_explorer/templates/student_explorer/about.html

```diff
@@ -9,13 +9,12 @@ <h1 class="sub-header">About Student Explorer</h1>
             identify students at risk of academic jeopardy, the tool enables advisors to create timely interventions
             directing students toward resources that may facilitate behavioral change and academic growth and success.
             For more information about how Student Explorer works and how to use it, please refer to the
-            <a href="https://documentation.its.umich.edu/student-explorer-general">Student Explorer documentation</a>.
+            <a href="https://documentation.its.umich.edu/student-explorer-general" target="_blank">Student Explorer documentation</a>.
         </p>
 
         <h2 class="list-header">Student Explorer Access</h2>
-        <p>Student Explorer is available to all professional staff/faculty advisors at the U-M Ann Arbor campus upon
-            request for access and includes information about all students on the Ann Arbor campus, including graduate
-            students. Students, including peer advisors, are not able to gain access to Student Explorer.
+        <p>Student Explorer is available to all professional staff/faculty advisors at the U-M Ann Arbor campus upon request for access and includes information about all students on the Ann Arbor campus who use Canvas, including graduate
+            students (note: some graduate programs do not use Canvas and those students will not be visible in Student Explorer). Learn more about <a href="https://documentation.its.umich.edu/student-explorer-access" target="_blank">Student Explorer access</a>.
```

Comment:
Maybe we can link the whole "Learn more ..." expression here, like we did in "About Cohorts"?